include/CL/sycl/math.hpp

```diff
@@ -0,0 +1,170 @@
+#ifndef TRISYCL_SYCL_MATH_HPP
+#define TRISYCL_SYCL_MATH_HPP
+
+/** \file
+    Implement a wrapper around OpenCL math operations
+    Joan.Thibault AT ens-rennes POINT fr
+    This file is distributed under the University of Illinois Open Source
+    License. See LICENSE.TXT for details.
+*/
+
+#include <cmath>
+
+namespace cl {
+namespace sycl {
+#define TRISYCL_MATH_WRAP(FUN) template<typename T> T FUN(T x) \
+  {return std::FUN(x);}
+#define TRISYCL_MATH_WRAP2(FUN) template<typename T> T FUN(T x, T y) \
+  {return std::FUN(x, y);}
```

Comment:
All the formatting of {} on all the following lines :-(
#define TRISYCL_MATH_WRAP2(FUN) template<typename T> T FUN(T x, T y) \
  { \
    return std::FUN(x, y); \
  }

For example. In Emacs the \ are aligned on the right, but this is OK like that.